src/components/PinInput/src/PinInput.vue

```diff
@@ -126,13 +126,13 @@ export default {
 		},
 
 		onInputPin(event, index) {
-			if (!event.data) {
+			const input = event?.data || event?.target?.value;
```

Comment:
Could you add some comments here about the data values and the reason for the conditional? Also any external references regarding why safari works a bit differently if they exist?

Reply:
Im just going to remove the conditional as per Kirills suggestion